src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -753,9 +789,15 @@ def main():
         ws_url = os.path.join(kb_base_url, "ws")
         ws = Workspace(ws_url, token=conf.token)
 
-        uploaded_obj_names, uploaded_obj_upas = _query_workspace_with_load_id_mass(
-            ws, workspace_id, load_id, list(wait_to_upload_objs.keys())
-        )
+        (uploaded_obj_names,
+         uploaded_obj_upas,
+         uploaded_assembly_objs_info,
+         uploaded_genome_objs_info) = _query_workspace_with_load_id_mass(ws,
+                                                                         workspace_id,
+                                                                         load_id,
+                                                                         list(wait_to_upload_objs.keys()),
+                                                                         assembly_objs_only=create_assembly_only)
+
```

Comment:
At this point the main function is 175 lines long, which is way too long. In a PR pretty soon we need to split this up into cohesive functions.

Reply:
coming up next.

Reply:
:+1: